bigbone/src/main/kotlin/social/bigbone/api/method/SuggestionMethods.kt

```diff
@@ -0,0 +1,43 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.entity.Suggestion
+
+/**
+ * Allows access to API methods with endpoints having an "api/vX/suggestions" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/suggestions/">Mastodon API documentation: methods/suggestions/</a>
+ */
+class SuggestionMethods(private val client: MastodonClient) {
+
+    private val suggestionsEndpointV2 = "/api/v2/suggestions"
+    private val suggestionsEndpointV1 = "/api/v1/suggestions"
+
+    /**
+     * Accounts that are promoted by staff, or that the user has had past positive interactions with, but is not yet following.
+     * @param limit to limit number of results
+     * @see <a href="https://docs.joinmastodon.org/methods/suggestions/#v2">Mastodon API documentation: methods/suggestions/#v2</a>
+     */
+    fun getSuggestions(limit: Int? = null): MastodonRequest<List<Suggestion>> {
+        return client.getMastodonRequestForList(
+            endpoint = suggestionsEndpointV2,
+            method = MastodonClient.Method.GET,
+            parameters = Parameters().apply {
+                append("limit", limit?.coerceIn(1, 80) ?: 40)
```

Comment:
This hides potentially invalid inputs for `limit`. I’d suggest to take #293 into account here already and add client-side validation to the start of this method, e.g.:

```kotlin
if (limit != null && limit > QUERY_RESULT_LIMIT) {
    throw IllegalArgumentException(
        "limit defined in Range must not be higher than $QUERY_RESULT_LIMIT but was $limit"
    )
}
```
where `QUERY_RESULT_LIMIT` would be defined as a const with value `80`.

The lower limit you introduce with `coerceIn(1, 80)` is also a very good idea though. This should probably be added to the top as well! 👍